src/main/groovy/edu/oregonstate/mist/locations/frontend/resources/LocationResource.groovy

```diff
@@ -0,0 +1,51 @@
+package edu.oregonstate.mist.locations.frontend.resources
+
+import com.codahale.metrics.annotation.Timed
+import edu.oregonstate.mist.api.AuthenticatedUser
+import io.dropwizard.auth.Auth
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+
+import javax.ws.rs.GET
+import javax.ws.rs.Path
+import javax.ws.rs.PathParam
+import javax.ws.rs.Produces
+import javax.ws.rs.WebApplicationException
+import javax.ws.rs.client.Client
+import javax.ws.rs.core.MediaType
+import javax.ws.rs.core.Response
+
+@Path("/locations")
+@Produces(MediaType.APPLICATION_JSON)
+class LocationResource {
+    private static final Logger LOGGER = LoggerFactory.getLogger(LocationResource.class)
+
+    private final Map<String, String> locationConfiguration
+    final Client client
+
+    LocationResource(Map<String, String> locationConfiguration, Client client) {
+        this.locationConfiguration = locationConfiguration
+        this.client = client
+    }
+
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    @Timed
+    @Path('{id}')
```

Comment:
consider specifying the format with a regular expression as [recommended](https://github.com/osu-mist/directory-api/pull/1#discussion-diff-48285445) and [implemented](https://github.com/osu-mist/directory-api/commit/cb6e712de800fa4dfcac65efe5709327f2cb0e34) in `directory-api`